### PR TITLE
CAT-1838 Separate events from other control bricks

### PR DESCRIPTION
**Pull request by robertriedl**

### CODE REVIEW

File: catroid/res/values/styles.xml [hunk 1]
```diff
@@ -443,6 +443,23 @@
...
     </style >
 
+    <style name="BrickContainer.Event" ></style >
+
+    <style name="BrickContainer.Event.Small" >
+        <item name="android:minHeight" >@dimen/brick_height_small</item >
+        <item name="android:background" >@drawable/brick_brown_1h_when_9</item >
+    </style >
+
+    <style name="BrickContainer.Event.MediumWhen9" >
```
Comment (loibi93) [inline catroid/res/values/styles.xml]: Why MediumWhen**9** ?

Comment (robertriedl) [inline catroid/res/values/styles.xml]: because of the file-names ;)

Comment (loibi93) [inline catroid/res/values/styles.xml]: The 9 stands for 9 Patch Graphik, no need to include it in the style name, this is just confusing ;D

Comment (robertriedl) [inline catroid/res/values/styles.xml]: just skimmed over it and didn't see that this isn't the file name xD

**Pull request closed without merging**